neo4j/templates/neo4j-config.yaml

```
@@ -126,6 +126,7 @@ data:
   dbms.routing.default_router: "SERVER"
   dbms.routing.client_side.enforce_for_domains: "*.{{ .Values.clusterDomain }}"
   dbms.routing.enabled: "true"
+  dbms.cluster.discovery.version: "V1_ONLY"
```

If we want users to be able to set this, I think we should be setting it in `values.yaml`, and fetching that value here. E.g. 
```suggestion
  dbms.cluster.discovery.version: "{{ default "V1_ONLY" .Values.discoveryVersion }}"
```
The `default` may be excessive if we put it in the packaged `values.yaml`.

---

Right that's how it works, I've added it in [d0e95a4](https://github.com/neo4j/helm-charts/pull/351/commits/d0e95a43de200543dd3ba10302ef425fba8e557a)